src/domain/storage/document/document.entity.ts

```diff
@@ -4,13 +4,19 @@ import { StorageBucket } from '../storage-bucket/storage.bucket.entity';
 import { MimeFileType } from '@common/enums/mime.file.type';
 import { AuthorizableEntity } from '@domain/common/entity/authorizable-entity';
 import { Tagset } from '@domain/common/tagset';
+import {
+  ENUM_LENGTH,
+  MID_TEXT_LENGTH,
+  SMALL_TEXT_LENGTH,
+  UUID_LENGTH,
+} from '@common/constants';
 
 @Entity()
 export class Document extends AuthorizableEntity implements IDocument {
   // omitting OneToOne decorator for createdBy to avoid circular dependency
   // needs a redesign to avoid circular dependency
-  @Column('char', { length: 36, nullable: true })
-  createdBy!: string;
+  @Column('char', { length: UUID_LENGTH, nullable: true })
```

Comment:
createdBy is mandatory in teh code, and I think it should also be here. We need to track who uploaded a document.

Reply:
We need to standardize the "creator". Currently, any 36-long string can be a creator.
In addition, if the code creates something who would be the creator?

Reply:
If the code creates something then we have to find a reasonable identifier to put in. E.g. for the user creation flow in bootstrap I put the actual user id in. But it should be mandatory for documents so we know who created it.